src/jobs/jobs.js

```diff
@@ -59,6 +61,7 @@ async function paginateSyncOrders(orders, page = 15) {
 		const ordersToSync = orders.slice(i, i + page);
 		const ids = await processSync(ordersToSync);
 		ordersIds.push(...ids);
+		await sleep(100);
```

Comment:
Why do we need to sleep ?

Reply:
Instead of sleeping the process, you could increase the time between each cronjob run.

Reply:
The sync process was causing the rateLimit to be exceeded, so I had to add sleep to give it a little time between requests. I removed it and placed fewer orders per page to see if we could avoid exceeding it.

Reply:
If that happens to me, another way to avoid the rate limit error would be to use a for loop instead of Promise.all. Doing it synchronously. The issue is that at a point of high concurrency, the service's performance could be affected.